backend/app/models/viz_config.py

```diff
@@ -0,0 +1,38 @@
+from datetime import datetime
+from typing import Optional, List
+
+from beanie import Document, View, PydanticObjectId
+from pydantic import Field, BaseModel
+
+from app.models.authorization import AuthorizationDB
+from app.models.listeners import ExtractorInfo, EventListenerJobDB
+from app.models.metadata import MongoDBRef
+from app.models.pyobjectid import PyObjectId
+from app.models.users import UserOut
+
+
+class VizConfigBase(BaseModel):
+    resource_id: PydanticObjectId
+    resource_type: str
```

Comment:
Since we are using the resource ref on the VizConfigDB, do we still need the `resource_id` and `resource_type`?

Reply:
I'll remove these, you are right they are no longer needed.